### PR TITLE
fix continuity count cc

### PR DESCRIPTION
**Pull request by 9000h**

### CODE REVIEW

File: src/stream.c [hunk 1]
```diff
@@ -837,14 +837,20 @@ int check_cc(adapter *ad)
...
 	if ((p = find_pid(ad->id, 8192)))
 		return 0;
 
 	for (i = 0; i < ad->rlen; i += DVB_FRAME)
 	{
 		b = ad->buf + i;
-		pid = (b[1] & 0x1f) * 256 + b[2];
+		pid = (b[1] & 0x1f) << 8;
+		pid |= b[2];
 		if (pid == 8191)
```
Comment (lars18th) [inline src/stream.c]: Hi @9000h ,

If this code improves how to get the pid number (I assume is more portable between different platforms), then update this MACRO too:
https://github.com/catalinii/minisatip/blob/e4ee5fa587c33605c4baa8eba75d38b15ab5af61/src/pmt.h#L35

Comment (9000h) [reply src/stream.c]: will see what @catalinii has to say about, and there are more places to

File: src/stream.c [hunk 2]
```diff
@@ -837,14 +837,20 @@ int check_cc(adapter *ad)
 	int pid;
 	int packet_no_sid = 0;
 	unsigned char *b;
+        
+	if (ad->buf[0] != 0x47) {
```
Comment (catalinii) [inline src/stream.c]: I am not sure this looks right, you are skipping the entire method for one broken packet.
I think is better if we log it and skip processing it.

Comment (9000h) [reply src/stream.c]: ok did remove that part

**Comment by 9000h** [comment]
you are right

**Pull request closed without merging**